api/client/system/info.go

```diff
@@ -214,6 +214,7 @@ func runInfo(dockerCli *client.DockerCli) error {
 	}
 
 	fmt.Fprintf(dockerCli.Out(), "Live Restore Enabled: %v\n", info.LiveRestoreEnabled)
+	fmt.Fprintf(dockerCli.Out(), "Default Isolation: %v\n", info.Isolation)
```

Comment:
Technically Linux supports multiple isolation modes too, just the only one used is default. I think splitting into platform specific here is something for a follow on PR as there's things the other way around which could be tidied up. 

As for where in the output, no preference. I can put it anywhere you like :smile: